web_console/dynamic-plugin/overview-dynamic-plugin.adoc

```diff
@@ -40,6 +42,12 @@ conster Header: React.FC = () => {
 
 * Avoid selectors that could affect markup outside of your plugins components, such as element selectors. These are not APIs and are subject to change. Using them might break your plugin. Avoid selectors like element selectors that could affect markup outside of your plugins components.
 * Provide valid JavaScript Multipurpose Internet Mail Extension (MIME) type using the `Content-Type` response header for all assets served by your plugin web server. Each plugin deployment should include a web server that hosts the generated assets of the given plugin.
+* You must build your plugin with Webpack using Webpack version 5+.
+* You must use link:https://nodejs.org/en/[`Node.js`] and link:https://yarnpkg.com/[`yarn`] to build and run your plugin.
```

Comment:
You could use npm I believe. yarn isn't strictly required.

@vojtechszocs keep me honest